src/pages/[[...slug]].tsx

```diff
@@ -29,6 +29,8 @@ export default function ResourcePage({ resource, globalElements }) {
     <Wrapper bannerData={globalElements.bannerData}>
       <Head>
         <title>{title}</title>
+        {/* todo: do all meta tags correctly, this fixes an error on news story */}
+        <meta property="og:url" content={path} />
```

Comment:
**MUST (or at least probably should eventually)**
I'm not sure this is going to work. `path` is not the current url in all cases. For listing pages, `path` is the url of the first page.

We can easily get what we need, but we'll need to make a few changes (including maybe naming, since it does feel like `path` is the current url) in `getStaticProps`.

Reply:
Heard chef. I've removed this for now with a hardcoded 'foo'. We will handle meta info properly in a follow up ticket.